crates/sui-json/src/tests.rs

```diff
@@ -905,10 +905,19 @@ fn test_string_vec_df_name_child_id_eq() {
         }],
     });
 
-    let layout = MoveTypeLayout::Struct(MoveStructLayout::WithFields(vec![MoveFieldLayout::new(
-        Identifier::from_str("labels").unwrap(),
-        MoveTypeLayout::Vector(Box::new(string_layout)),
-    )]));
+    // TODO(tzakian)[layout] double-check this....
```

Comment:
Is this a TODO for before or after this PR lands?

Reply:
This was for before this PR (as I was making it) but forgot to remove it. So will do that now :)